scripts/SDM/extentToBbox.yml

```diff
@@ -1,13 +1,68 @@
 script: extentToBbox.R
```

Comment:
I'm confused that we have extentToBbox here and shpToBbox in the extent folder.
This one can also take a shape file. We should choose : one step that can handle everything, but with some parameters that you have to know which ones you need, or a step per type of input, where all the inputs of the step are always needed. I prefer the latter, since it creates less confusion than empty params...

Reply:
extentToBox replace shpToBBox: it can take a shapefile, dataframe or a list of coordinates